docs/src/documentation/04-design-patterns/01-progressive-disclosure.mdx

```diff
@@ -188,12 +188,12 @@ so it's best to hide it when it's not being used.
 to show that there is additional context available behind the text.
 They're a basic way of keeping information present but hidden.
 
-You can see an example in the [pricing table](/components/structure/pricingtable/) below.
-The features are presented simply for comparison,
-but each has a tooltip for explaining additional details.
-This can separate more complex information, like lists.
+You can see an example in the list below.
+The features are presented briefly to ease comparison, for example.
```

Comment:
`for example` seems like useless here ;)

Reply:
The `for example` is for `to ease comparison`, because features can be presented briefly for other reason, like lack of space. But I can remove it. I even think it looks better without it 😄